x/mongo/driver/operation.go

```diff
@@ -1120,7 +1125,7 @@ func (op Operation) createMsgWireMessage(ctx context.Context, maxTimeMS uint64,
 		flags |= wiremessage.ExhaustAllowed
 	}
 
-	info.requestID = wiremessage.CurrentRequestID()
```

Comment:
Optional: Are any uses of `CurrentRequestID` safe? Should we remove that function completely and only call `NextRequestID`?

Reply:
Good Idea, I would guess not since the requestID is a global variable. Done.